src/applications/search/containers/SearchApp.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { withRouter } from 'react-router';
 import { connect } from 'react-redux';
+import moment from 'moment-timezone';
```

Comment:
**Consider**: We should be moving away from using `moment` and `moment-timezone` as it is a much larger package/dependency than others like `date-fns`. If it's not a simple refactor to use another library, don't worry about it; just a "consider" here.

Reply:
good feedback, I'll try to use date-fns